### DISCUSSION TITLE
How can I build a benchmark for Image Segmentation?

### DISCUSSION THREAD

**Discussion by KirigiriSuzumiya** [ContinualAI/avalanche#1626] [Q&A]
I don't know whether it's possible to train an Image Segmentation model on avalanche, as I find that most of the benchmarks are build on classification tasks. For instance, how can I train a U-Net with a benchmark whose output layer is not a single class, but like [w,h]?

**Comment by Ali-Olliek** [comment]
Hello.
 
Did you find a way to achieve this?

**Reply by KirigiriSuzumiya** [reply]
yes, you can take a look at my code [here](https://github.com/KirigiriSuzumiya/CL_for_ImageSegmentation/tree/main/experience). Most of the work is on building the benchmark

**Reply by Ali-Olliek** [reply]
Thank you!
